# speeds_map: {0.0: [1100.0, 1000.0], 0.5: [1700, 1500], 2.0: [2000, 1800]}
# angle_map: {0.0: 1.22, 1.0, 0.87, 1.5: 0.698, 5.0: 0.349}

# speeds_map: [[distance: [top_left_speed, top_right_speed, bottom_left_speed, bottom_right_speed]], ...]
speeds_map: [[0.0, [110.0, 100.0, 110.0, 100.0]], [0.5, [170.0, 150.0, 200.0, 180.0]], [2.0, [200.0, 180.0, 300.0, 150.0]]]
angle_map: [[0.0, 1.22], [1.0, 0.87], [1.5, 0.698], [5.0, 0.349]]
delay_after_shooting: 0.5